package.json

```diff
@@ -0,0 +1,65 @@
+{
+  "name": "maker-docker-poa",
+  "version": "0.0.1",
+  "description": "Maker SAI compiler and deployer",
+  "author": "Keydonix <team@keydonix.com>",
+  "repository": "http://github.com/keydonix/maker-docker-poa",
+  "main": "output/index.js",
+  "types": "output/index.d.ts",
+  "module": "output/index.js",
+  "browser": "output/browser.js",
+  "license": "GPL-3.0",
+  "directories": {
+    "src": "source"
+  },
+  "scripts": {
+    "build:source": "npx tsc",
+    "build:contracts": "npm run fetch:sai && node output/deployment/compileContracts.js",
+    "build:interfaces": "npm run fetch:sai && node output/tools/generateContractInterfaces.js",
+    "build": "npm run build:source && npm run build:contracts && npm run build:interfaces",
+    "clean": "rimraf output",
+    "clean-start": "npm run clean && npm run build",
+    "docker:build": "docker build . -t keydonix/maker-docker-poa",
+    "preversion": "npm run build",
+    "version": "git add source/libraries/ContractInterfaces.ts",
+    "release": "npm version prerelease && npm publish",
+    "fetch:sai": "bash fetch-sai-contracts.sh"
+  },
+  "dependencies": {
+    "@types/chai": "4.0.4",
+    "@types/chai-as-promised": "7.1.0",
+    "@types/get-port": "^3.2.0",
+    "@types/mkdirp": "0.5.1",
+    "@types/mocha": "2.2.43",
+    "@types/node": "9.6.7",
+    "async-file": "2.0.2",
+    "async-mkdirp": "1.2.0",
+    "binascii": "0.0.1",
+    "bn.js": "4.11.8",
+    "chai": "4.1.2",
+    "chai-as-promised": "7.1.1",
+    "crypto-promise": "^2.0.0",
+    "ethereumjs-testrpc": "6.0.3",
+    "ethjs-abi": "0.2.1",
+    "ethjs-account": "0.1.4",
+    "ethjs-contract": "0.1.9",
+    "ethjs-provider-http": "0.1.6",
+    "ethjs-query": "0.3.1",
+    "ethjs-rpc": "0.1.8",
+    "ethjs-signer": "0.1.1",
+    "fs-readfile-promise": "3.0.0",
+    "get-port": "3.2.0",
+    "mocha": "3.5.3",
+    "path": "0.12.7",
+    "recursive-readdir": "2.2.1",
+    "copy-dir": "0.3.0",
+    "rimraf": "2.6.2",
+    "replace": "0.3.0"
+  },
+  "devDependencies": {
+    "solc": "0.4.20",
+    "source-map-support": "0.5.0",
+    "ts-node": "3.3.0",
+    "typescript": "2.6.1"
```

Comment:
Consider bumping all of these dependencies to latest so we aren't starting with old libraries out of the gate.  For example, typescript is up to 2.8.x.